firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/send/ReportQueue.java

```diff
@@ -115,6 +117,24 @@ TaskCompletionSource<CrashlyticsReportWithSessionId> enqueueReport(
     }
   }
 
+  public void flushScheduledReportsIfAble() {
+    CountDownLatch latch = new CountDownLatch(1);
+    new Thread(
+            () -> {
+              TransportRuntime.getInstance()
+                  .getUploader()
+                  .upload(DataTransportCrashlyticsReportSender.transportContext, 1, () -> {});
+              latch.countDown();
+            })
+        .start();
+    try {
+      //noinspection ResultOfMethodCallIgnored best effort only
+      latch.await(2, TimeUnit.SECONDS);
+    } catch (InterruptedException unused) {
+
+    }
```

Comment:
You shouldn't swallow this exception: https://engdoc.corp.google.com/eng/doc/devguide/java/practices/interruption.md#interruptedexception

Instead maybe mimic awaitUninterruptibly from guava https://github.com/google/guava/blob/94786f7c419041068ad48f04fe3d9fa4a266b6dd/guava/src/com/google/common/util/concurrent/Uninterruptibles.java#L56-L72

Reply:
Done